src/renderer/renderScene.ts

```diff
@@ -166,11 +134,10 @@ export function renderScene(
       context.lineWidth = lineWidth;
       context.setLineDash(initialLineDash);
     });
-    resetZoom(context);
+    context.translate(-sceneState.scrollX, -sceneState.scrollY);
```

Comment:
The initial translate got reset via context.reset, which is not the most expected behavior